### ISSUE TITLE
Acceptance test: UC-47

### ISSUE THREAD

**Issue by OmniaGo** [rThamb/soen390-schoolmap#123]
**UC-47, As a user, I want to be able to get directions from one room to another room on different floors**

**Acceptance Criteria**

1. Given that the user is on the “New Route” page.
2. The user selects a start and destination point on different floors.
3. The user selects the floor number of the start point
4. A map of the floor will be displayed including pins on the start and end points and a path between them. 
5. The user selects the floor number of the end point
6. A map of the floor will be displayed including pins on the start and end points and a path between them

**Procedures**

1. Launch the app
2. Click on the hamburger button situated at the top left hand corner of the screen.
3. Select “New Route”
4. Set start to : HB840
5. Set destination to: HB922
6. Click on the button “Get Directions”
7. Select floor 8 in dropdown
8. Then, select floor 9 in dropdown

**Expected Output**
By selecting the start floor, in this case floor 8, a blue line will be displayed on the map showing the path of the selected start point to the selected end point on that start floor displayed pins on their location initialized with  for start and for end. Then, by selecting the end floor, in this case floor 9, both start and end points will have a red pins displayed on their location initialized with  for start and for end.

**Comment by mosaliza** [comment]
![Screenshot_20200409-132810_ConcordiaGo](https://user-images.githubusercontent.com/35632341/78923318-048fb080-7a66-11ea-92f9-8149099d56ac.jpg)
Not working!

**Comment by MNezameddin** [comment]
Step 1: check your settings to see which path the app will select to direct you
![Screenshot_20200409-134446.png](https://images.zenhubusercontent.com/5e1a89f389cf347b9596bb59/9c05b676-7c5d-4632-bf9c-2819e579afd5)

Step2: Go to the new route page and select HB840 as your starting location and HB922 as your destination
![Screenshot_20200409-134816.png](https://images.zenhubusercontent.com/5e1a89f389cf347b9596bb59/f9f34177-22e9-4e75-ab7b-8ddbb89a9e03)

Step 3: click on get directions button and you'll be navigated to the first floor of Hall
![Screenshot_20200409-134416.png](https://images.zenhubusercontent.com/5e1a89f389cf347b9596bb59/72c80400-de35-4a40-93ec-3b735b4ac121)

Step 4: click on the floor number at the top of the map and select the 8th & 9th floor to show the path (in my case since the settings is set all to true, it will automatically give me the elevators to use as a route)
![Screenshot_20200409-134428.png](https://images.zenhubusercontent.com/5e1a89f389cf347b9596bb59/2c84c349-bbca-4d27-a42c-605acb378f3a)

![Screenshot_20200409-134437.png](https://images.zenhubusercontent.com/5e1a89f389cf347b9596bb59/d03700a3-b7ca-4bae-b203-bad283b29f61)

**Comment by mosaliza** [comment]
Did you try it in android phones? Get direction does not show anything here after I do the path settings and set the start and ending points HB840 amd HB922.
![Screenshot_20200409-135931_ConcordiaGo](https://user-images.githubusercontent.com/35632341/78926022-5e927500-7a6a-11ea-9732-d9c5b8548d47.jpg)

**Comment by MNezameddin** [comment]
Yes, might have to do with the fact that we are using an android device. even for other tests, all my replies and screenshots were done on a pixel 3a XL. The sdk generated might work better for android for some reason, but when testing our code directly on an emulator, IOS and Android were both working.

**Comment by JenniferAbouJaoudi** [comment]
[Screen Recording 2020-04-09 at 8.56.28 PM.zip](https://github.com/rThamb/soen390-schoolmap/files/4459310/Screen.Recording.2020-04-09.at.8.56.28.PM.zip)


@mosaliza

**Comment by MNezameddin** [comment]
Please view the following video within the ZIP.

[UC-47.zip](https://github.com/rThamb/soen390-schoolmap/files/4473156/UC-47.zip)

Device Specs:

![Device_Specifications](https://user-images.githubusercontent.com/39063206/79186782-90983400-7de8-11ea-81d7-0d522e78bf7a.png)
